engines/bastion_katello/app/assets/javascripts/bastion_katello/environments/details/views/environment-docker.html

```diff
@@ -0,0 +1,31 @@
+<div data-extend-template="layouts/details-nutupane.html">
+
+  <div data-block="header"></div>
+  <div data-block="selection-summary"></div>
+
+  <div class="well" data-block="filters">
```

Comment:
That's cool, we can discuss and then change in one place (i.e. bastion) if necessary.